### ISSUE TITLE
Sync problem when used with iCloud

### ISSUE THREAD

**Issue by xshuai1006** [ivan-lednev/obsidian-day-planner#120]
I use obsidian in Windows 10 and two iOS equipment（iPhone and iPad) and I use day planner plugin in command mode;
I installed day planner, calendar, templater and many other plugins; I created a diary template with templater, than I use calendar plugin to create daily note appled withe the diary template, then I link today's day planner with current note with command and edit the note.
And there have been two problems with synchronization when in use with iCloud (I checkouted the official obsidian forum list, one obsidian developer said that when disabled the day planner plugin, the sync problem gone, and I made a test with a note without linked to day planner, and no sync problem)

1. **The input is restored**：open a note on Windows, then open the same note on an iOS device, and then edit the note on windows, and I find that the edited texts are restored as the opened note on the iOS device;
2. **Many copies are created**。

I will do more test laterly to test more different situations.

Sorry for my poor English, hope you guys can understand my description.

**Comment by xshuai1006** [comment]
Today's test: unchecked "Complete past planner items" and "mermaid Gantt", and no iCloud sync problem so far.

**Comment by sjsonneman** [comment]
I'm not 100% sure this is the same problem. Here is what happened to me. 
I created a day planner file on my Macbook and put in some times
I also installed the kanban plugin
I closed the lid and went to work
I opened the lid to my MacBook at work (obsidian was still open)
The day planner file was empty
the creation date was the present time (not the time at home when I created and populated it)
I opened obsidian on my iPhone
the file was there with the contents that I entered
I checked back on my Mac, the day planner file is not updating or syncing with my phone
I restarted obsidian on my Mac and no change
I opened iCloud and the Day Planner folder says "waiting to update". Opening the day planner file shows that the day planner file for today says "error"
in the meantime obsidian has updated the files in the day planner file and created a copy of the file "Day Planner 2022-04-06 2". Both files have the present as creation time. Both files are empty 
meanwhile the iPhone file remains unchanged
iCloud "files" app on my phone shows the original day planner file unchanged from this morning
I edited the morning dsayplanner file on my phone and it updated on the iCloud files app on my phone but on the Mac the file showed the message "waiting to update" and then error
turning off day planner plugin on my mac meant that the Mac day planner files (empty) updated and now the iPhone ones reflect the Mac ones and I lost my work from this morning. Fortunately I backed it up before the above experimentation. 
The Mac ones may have been the ones kept as they were more recently created?
So it seems that this plugin is somehow affecting iCloud syncing which seems to be similar to what xshuai1006 is reporting
